### ISSUE TITLE
is it detected?

### ISSUE THREAD

**Issue by suciuale** [Teklad/csgo-horker#42]
ik that valve doesnt require root permision but can it be detected?

**Comment by Teklad** [comment]
Short answer. No. Valve has no access to processes ran by root so they
can't scan the memory of the external.

I don't play much myself but I'm sure some other users can verify.

**Comment by ddjem87** [comment]
playing from 1/2 weeks now i think, no problems at all just be aware of overwatch ban its when you go obvious. Pretend to be legit and dont shoot through the wall and check for updates here does some files got updated even if is 1 file i am always doing it to be sure being undetected. (great work who made this cheat i love it)

**Comment by Teklad** [comment]
Thank you for the support!

**Comment by w3bn00b** [comment]
i just want to tell sig scan is possible even if its running on root,
change signature(add junk or smth...) to make sure you are safe

**Comment by ddjem87** [comment]
how can i add junk code is there .sln somewhere in csgo-horker-master?

**Comment by kusayuzayushko** [comment]
would be nice to have a short tutorial on what files to change and some "junk" examples. For those who are not familiar with programming.

**Comment by ddjem87** [comment]
you dont need to be a programmer to know how to add junk code its the same with pasting but here you just need to open all the .cpp files and add them one by one(thats alot of work)

watch a tutorial how to paste there is so many tutorials in youtube and download the source code from releases.

**Comment by Teklad** [comment]
@w3bn00b this is incorrect.  They literally can't scan the memory of the process so sig scanning is impossible.  At best they can try to match process names.

All you'd really have to do is change the name of the process.  Sig scanning is possible for **Internal** Linux cheats because you're injecting directly into a process.... so it's not truly 100% root.

**Comment by troniac** [comment]
Yes they are looking for names or patterns!
But you can use nvidias overlay to draw on OpenGL Games. ;-)

New storm is coming signed modules have arrived last month. They will now be able to check their signed modules. We are investigating but it looks quiet interesting, this doesn't break the cheat community but makes things complicated.

Junk Code doesn't prevent VAC!
